db/Seeders/RoleSeeder.cs

```diff
@@ -0,0 +1,175 @@
+                Permission.SET_DEFAULT_HOME_SCREEN,
+                Permission.ADD_EDIT_OWN_NOTES_ONLY
+            };
+
+            var trainerPermissions = new List<string>
+            {
+                Permission.VIEW_DASHBOARD,
+                Permission.VIEW_OWN_SCHEDULE,
+                Permission.VIEW_OTHER_JUDGE_SCHEDULE,
+                Permission.VIEW_OTHER_CALENDARS,
+                Permission.VIEW_RESERVED_JUDGMENTS,
+                Permission.VIEW_COURT_LIST,
+                Permission.RETRIEVE_OWN_COURT_LIST,
+                Permission.SEARCH_WITHIN_COURT_LIST,
+                Permission.FILTER_WITHIN_COURT_LIST,
+                Permission.SEARCH_FOR_COURT_LISTS_BY_COURTROOM_AND_DATE,
+                Permission.SEARCH_FOR_COURT_FILES,
+                Permission.ADVANCED_SEARCH_FOR_COURT_FILES,
+                Permission.SORT_SEARCH_RESULTS,
+                Permission.VIEW_COURT_FILE,
+                Permission.VIEW_YOUTH_CRIMINAL_FILE,
+                Permission.VIEW_SUMMARY,
+                Permission.VIEW_ADJUDICATOR_RESTRICTIONS,
+                Permission.VIEW_ACCUSED,
+                Permission.VIEW_PARTIES,
+                Permission.VIEW_CHILDREN,
+                Permission.VIEW_CASE_DETAILS,
+                Permission.VIEW_APPEARANCES,
+                Permission.VIEW_SENTENCE_ORDER_DETAILS,
+                Permission.VIEW_SINGLE_DOCUMENT,
+                Permission.VIEW_MULTIPLE_DOCUMENTS,
+                Permission.VIEW_SIDE_BY_SIDE_DOCUMENTS,
+                Permission.ADD_EDIT_OWN_DOCUMENT_ANNOTATIONS_ONLY,
+                Permission.VIEW_QUICK_LINKS,
+                Permission.SET_DEFAULT_HOME_SCREEN,
+                Permission.ADD_EDIT_OWN_NOTES_ONLY
+            };
+
+            var roles = new List<Role>
+            {
+                new() {
+                    Name = "Admin",
+                    Description = "Role for JASPER system administrators",
+                    PermissionIds = [..
+                        permissions
+                            .Where(p => adminPermissions.Contains(p.Code))
+                            .Select(p => p.Id)
+                    ]
+                },
+                new() {
+                    Name = "Trainer",
+                    Description = "Role for JASPER trainers",
+                    PermissionIds = [..
+                        permissions
+                            .Where(p => trainerPermissions.Contains(p.Code))
+                            .Select(p => p.Id)
+                    ]
+                },
+                new() {
+                    Name = "Judge",
+                    Description = "Role for provincial judge",
+                    PermissionIds = [..
+                        permissions
+                            .Where(p => judgePermissions.Contains(p.Code))
+                            .Select(p => p.Id)
+                    ]
+                }
+            };
```

Comment:
I think you should more the defaults (default role names and descriptions) to the Role class, in the same way the defaults for the Groups and Permissions are defined in their owning entity.